lib/infra/infra-stack.ts

```diff
@@ -740,11 +755,20 @@ export class InfraStack extends Stack {
       });
     }
 
-    const cfnInitConfig: InitElement[] = [
+    if ((nodeType === 'data' || nodeType === 'single-node') && this.useInstanceBasedStorage) {
```

Comment:
Any reason we just want this for data notes and not others?

Reply:
All the work is done by data nodes. Manager and seed nodes only store cluster state information and do not do any data processing.